flamby/datasets/fed_camelyon16/benchmark.py

```diff
@@ -73,7 +73,7 @@ def main(num_workers_torch, log=False, log_period=10, debug=False, cpu_only=Fals
         if use_gpu:
             m = m.cuda()
         loss = BaselineLoss()
-        optimizer = optim.Adam(m.parameters(), lr=LR)
+        optimizer = optim.AdamW(m.parameters(), lr=LR)
```

Comment:
Could you use it by calling Optimizer from flamby.datasets.fed_camelyon16 instead ?